bentoml/repository/__init__.py

```diff
@@ -147,11 +153,31 @@ def add(self, bento_name, bento_version):
 
     def get(self, bento_name, bento_version):
         # Return s3 path containing uploaded Bento files
-        raise NotImplementedError
+
+        object_name = self._get_object_name(bento_name, bento_version)
+
+        try:
+            response = self.s3_client.generate_presigned_url(
+                'get_object',
+                Params={'Bucket': self.bucket, 'Key': object_name},
+                ExpiresIn=self._expiration,
+            )
+        except Exception as e:
+            raise BentoMLRepositoryException(
```

Comment:
I think generate_presigned_url returns None if it fails?

Could you make it check if the response is None and raise the same exception in that case?

Reply:
I'm following https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-presigned-urls.html
So it will raise an exception in that case and the response is always a valid URL.

Reply:
yes the doc says``` :return: Presigned URL as string. If error, returns None.``` it's probably better to give the user an error with message in that case, rather than return None

Reply:
Oh, I find the sentence you mentioned in the doc. It's for the demo function `create_presigned_url `,  rather than the `generate_presigned_url` function. Apart from that, yeah I think it's better to provide an error than returning None.

Reply:
Got it, that make sense